### DISCUSSION TITLE
Dúvida sobre HTML

### DISCUSSION THREAD

**Discussion by P4dro-Dev** [BackandDeveloper/Projeto-Front-and-sobre-T-quio#3] [Q&A] [answered]
Como faço para deixar meu HTML mais organizado?

**Comment by iegoassis** [comment, marked as answer]
Para deixar seu HTML mais organizado, siga estas dicas:

Estrutura
1. *Use indentação*: Utilize espaços ou tabs para indentar o código.
2. *Separe seções*: Divida o código em seções lógicas (cabeçalho, corpo, rodapé).
3. *Ordene elementos*: Coloque elementos semelhantes juntos (links, scripts, estilos).

Nomeação
1. *Use nomes descritivos*: Escolha IDs e classes claras e concisas.
2. *Evite nomes genéricos*: Em vez de "botão", use "botão-enviar".
3. *Use prefixos*: Para classes, use prefixos como "header-", "footer-".

Semântica
1. *Use elementos semânticos*: Em vez de `<div>`, use `<header>`, `<nav>`, `<main>`, `<section>`, `<footer>`.
2. *Atributos*: Utilize atributos como `alt`, `title` e `aria-` para acessibilidade.
3. *Estrutura de lista*: Use `<ul>`, `<ol>`, `<li>` para listas.

Organização
1. *Crie um índice*: Utilize comentários para marcar seções.
2. *Separe código*: Coloque scripts e estilos em arquivos separados.
3. *Use ferramentas*: Utilize editores de código com recursos de formatação automática.

Ferramentas
1. HTMLLint
2. HTMLTidy
3. Prettier
4. Visual Studio Code
5. Sublime Text

Boas práticas
1. *Mantenha atualizado*: Siga as últimas especificações do HTML5.
2. *Teste*: Verifique compatibilidade entre navegadores.
3. *Documente*: Adicione comentários para explicar o código.

Recursos adicionais
1. W3C ((link unavailable))
2. MDN Web Docs ((link unavailable))
3. HTML5 Rocks ((link unavailable))
4. CodePen ((link unavailable))
5. Stack Overflow ((link unavailable))

A organização do código HTML melhora:

- Legibilidade
- Manutenibilidade
- Acessibilidade
- Desempenho
- Compatibilidade
